apps/src/lib/levelbuilder/CollapsibleEditorSection.jsx

```diff
@@ -1,29 +1,37 @@
 import PropTypes from 'prop-types';
 import React, {useState} from 'react';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
-import styleConstants from '@cdo/apps/styleConstants';
+import style from './levelbuilder.module.scss';
+import classnames from 'classnames';
 
 export default function CollapsibleEditorSection(props) {
   const [collapsed, setCollapsed] = useState(props.collapsed || false);
 
   const {title, fullWidth} = props;
-  const editorsStyle = {
-    ...styles.editors,
-    width: fullWidth ? null : styleConstants['content-width'],
-  };
 
   return (
     <div>
-      <div style={styles.header}>
-        <h2 onClick={() => setCollapsed(!collapsed)} style={styles.title}>
-          <FontAwesome
-            icon={collapsed ? 'expand' : 'compress'}
-            style={styles.icon}
-          />
-          {title}
+      <div className={style.collapsibleEditorSectionHeader}>
+        <h2 className={style.collapsibleEditorSectionTitle}>
+          <button
```

Comment:
This pattern is [explicitly recommended](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-interactions.md#case-this-is-a-heading-that-expandscollapses-content-on-the-package) by the linter documentation